node/sc/vt_recovery_test.go

```diff
@@ -632,7 +632,7 @@ func TestMultiOperatorRequestRecovery(t *testing.T) {
 // prepare generates dummy blocks for testing value transfer recovery.
 func prepare(t *testing.T, vtcallback func(*testInfo)) *testInfo {
 	// Setup configuration.
-	config := &SCConfig{}
+	config := &SCConfig{ServiceChainDefaultGasLimit: 5000000}
```

Comment:
It is still 5000000. Is it intended?

Reply:
I deleted this field because don't need it (can replace by default value). Thanks for review.